src/components/Popover/Popover.tsx

```diff
@@ -40,10 +40,11 @@ interface PopoverContentProps extends RadixPopover.PopoverContentProps {
 
 const MenuPanel = styled(GenericMenuPanel)<{ showClose?: boolean; showArrow?: boolean }>`
   display: block;
-  padding: ${({ theme }) => theme.click.popover.space.y} ${({ theme }) => theme.click.popover.space.y};
-  background-color: ${({ theme }) => theme.click.popover.color.panel.background}
-  border: 1px solid ${({ theme }) => theme.click.popover.color.panel.stroke}
-  border-radius: ${({ theme }) => theme.click.popover.radii.all}
+  padding: ${({ theme }) => theme.click.popover.space.y} ${({ theme }) => theme.click.popover.space.x};
```

Comment:
We can add this in a single line like below
`padding: ${({ theme }) => `${theme.click.popover.space.y} ${theme.click.popover.space.x}`};`